src/Files.App/UserControls/AddressToolbar.xaml

```diff
@@ -35,6 +35,7 @@
 				<ResourceDictionary>
 					<SolidColorBrush x:Key="ButtonBorderBrushDisabled" Color="Transparent" />
 					<SolidColorBrush x:Key="ButtonBackgroundDisabled" Color="Transparent" />
+					<SolidColorBrush x:Key="AppBarButtonBackgroundPointerOver" Color="{ThemeResource ButtonBackgroundPointerOver}" />
```

Comment:
```suggestion
```

Reply:
This makes the toggle button have the same background as the buttons.